CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_netsuite v0.14.0
+
+[PR #132](https://github.com/fivetran/dbt_netsuite/pull/132) includes the following updates:
+
+## Breaking Changes (Full refresh required after upgrading)
+- Partitioned models have had the `partition_by` logic adjusted to include a granularity of a month. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error some users have experienced do to over partitioning by day. Therefore, adjusting the partition to a month granularity will increase the partition windows and allow for more performant querying. This change was applied to the following models:
```

Comment:
```suggestion
- Partitioned models have had the `partition_by`  granularity adjusted from `day` to `month`. This change should only impact BigQuery warehouses and was applied to avoid the common `too many partitions` error users have experienced due to over-partitioning by day. Therefore, adjusting the partition to a monthly granularity will increase the partition windows and allow for more performant querying. 
- This change was applied to the following models:
```